amqp/subscriber.go

```diff
@@ -28,7 +30,7 @@ type Subscriber interface {
 	SubscribeDeviceUplink(appID, devID string, handler UplinkHandler) error
 	SubscribeAppUplink(appID string, handler UplinkHandler) error
 	SubscribeUplink(handler UplinkHandler) error
-	ConsumeUplink(queue string, handler UplinkHandler) error
+	ConsumeMessages(queue string, uplinkHandler UplinkHandler, eventHandler DeviceEventHandler) error
```

Comment:
How so harder to manage? The original `ConsumeUplink()` was pretty straight forward. The difference would be the routing key. AMQP is _only_ used in performance critical scenarios. I don't want to nop-handle all device events in _most_ integrations because it's easier to implement.